ta/pkcs11/user_ta.mk

```diff
@@ -1 +1,11 @@
 user-ta-uuid := fd02c9da-306c-48c7-a49c-bbd827ae86ee
+
+all: ci-checks
+
+.PHONY: ci-checks
+ci-checks: pkcs11-ta-ci-verify-helpers
```

Comment:
Why this intermediate target?

Reply:
A place to chain more checks in future if needed -- can drop the level now if wanted.